### PR TITLE
Reduce number of srv in Ros controller (doc)

### PR DESCRIPTION
**Pull request by DavidMansolino**
Linked to omichel/webots#5593

https://www.cyberbotics.com/doc/reference/ros-api?version=enhancement-ros-service-types

### CODE REVIEW

**Comment by DavidMansolino** [comment]
👍

**Comment by DavidMansolino** [comment]
(not sure if I should target develop of webots8.6.0)

**Comment by stefaniapedrazzi** [comment]
This should be merged in webots8.6.0.

**Comment by DavidMansolino** [comment]
thank you

File: reference/ros-api.md [hunk 1]
```diff
@@ -294,75 +309,77 @@ for more information about the value of `robot_unique_name`).
...
+| [/`<`device\_name`>`/set\_channel](receiver.md#wb_receiver_set_channel)                    | service       | [webots\_ros::set\_int](#common_services)      |                                                                                        |
+| [/`<`device\_name`>`/get\_channel](receiver.md#wb_receiver_set_channel)                    | service       | [webots\_ros::get\_int](#common_services)      |                                                                                        |
+| [/`<`device\_name`>`/get\_queue\_length](receiver.md#wb_receiver_get_queue_length)         | service       | [webots\_ros::get\_int](#common_services)      |                                                                                        |
+| [/`<`device\_name`>`/next\_packet](receiver.md#wb_receiver_get_queue_length)               | service       | [webots\_ros::set\_bool](#common_services)     |                                                                                        |
```
Comment (stefaniapedrazzi) [inline reference/ros-api.md]: Is set_bool correct or should it be get_bool?

File: reference/ros-api.md [hunk 2]
```diff
@@ -9,184 +9,199 @@ look at the [User
...
+| [/`<`device\_name`>`/set\_alpha](display.md#wb_display_set_color)       | service       | [webots\_ros::set\_float](#common_services)   |                                                                                     |
+| [/`<`device\_name`>`/set\_opacity](display.md#wb_display_set_color)     | service       | [webots\_ros::set\_float](#common_services)   |                                                                                     |
+| [/`<`device\_name`>`/set\_font](display.md#wb_display_set_color)        | service       | webots\_ros::display\_set\_font               | string font<br/>int32 size<br/>uint8 antiAliasing<br/>---<br/>int8 success          |
+| [/`<`device\_name`>`/set\_attach\_camera](display.md#wb_display_attach_camera) | service | [webots\_ros::set\_string](#common_services) |                                                                                     |
+| [/`<`device\_name`>`/set\_detach\_camera](display.md#wb_display_attach_camera) | service | [webots\_ros::set\_bool](#common_services)   |                                                                                     |
```
Comment (stefaniapedrazzi) [inline reference/ros-api.md]: set_bool or get_bool?

File: reference/ros-api.md [hunk 3]
```diff
@@ -294,75 +309,77 @@ for more information about the value of `robot_unique_name`).
...
+| [/supervisor/simulation\_quit](supervisor.md#wb_supervisor_simulation_quit)                                           | service       | [webots\_ros::set\_int](#common_services)                            |                                                                                                                                                                                                                                                                   |
+| [/supervisor/simulation\_revert](supervisor.md#wb_supervisor_simulation_revert)                                       | service       | [webots\_ros::set\_bool](#common_services)                           |                                                                                                                                                                                                                                                                   |
```
Comment (stefaniapedrazzi) [inline reference/ros-api.md]: set_bool or get_bool?

File: reference/ros-api.md [hunk 4]
```diff
@@ -294,75 +309,77 @@ for more information about the value of `robot_unique_name`).
...
+| [/supervisor/simulation\_reset\_physics](supervisor.md#wb_supervisor_simulation_reset_physics)                        | service       | [webots\_ros::set\_bool](#common_services)                           |                                                                                                                                                                                                                                                                   |
```
Comment (stefaniapedrazzi) [inline reference/ros-api.md]: set_bool or get_bool?

File: reference/ros-api.md [hunk 5]
```diff
@@ -294,75 +309,77 @@ for more information about the value of `robot_unique_name`).
...
-| [/supervisor/movie\_stop\_recording](supervisor.md#wb_supervisor_movie_start_recording)                               | service       | webots\_ros::supervisor\_movie\_stop\_recording                      | uint8 ask<br/>---<br/>int8 success                                                                                                                                                                                                                                |
+| [/supervisor/movie\_stop\_recording](supervisor.md#wb_supervisor_movie_start_recording)                               | service       | [webots\_ros::set\_bool](#common_services)                           |                                                                                                                                                                                                                                                                   |
```
Comment (stefaniapedrazzi) [inline reference/ros-api.md]: set_bool or get_bool?

**Comment by DavidMansolino** [comment]
I agree it would probably be better to use get_bool, and use the returned bool to indicate if the success or not.

**Comment by DavidMansolino** [comment]
👍

**Pull request merged**